### PR TITLE
Add supercharger dashboard

### PR DESCRIPTION
**Pull request by Dulanic**
Adding a supercharger dashboard. I added to doc also, wasn't positive if this would also add to the teslamate page? Please direct if another change needs to be added to create a new dashboard. Credit to Pankey on discord for this.

image

### CODE REVIEW

**Comment by adriankumpf** [comment]
Cool idea! 

>  I added to doc also, wasn't positive if this would also add to the teslamate page? 

Yup, it's added. At compile time all dashboards in `grafana/dashboards` are automatically included. So adding/removing/renaming dashboards does not need additional changes.

File: grafana/dashboards/supercharges.json [hunk 1]
```diff
@@ -0,0 +1,1067 @@
...
+        "ymin": null
+      },
+      "tableColumn": "",
+      "targets": [
+        {
+          "format": "time_series",
+          "group": [],
+          "metricColumn": "none",
+          "rawQuery": true,
+          "rawSql": "SELECT start_date as \"time\", charge_energy_added\nFROM charging_processes\nWHERE \n  $__timeFilter(end_date) AND \n  car_id = $car_id AND     \n   (charge_energy_added * 60 / NULLIF (duration_min, 0)) > 20 AND\n  ('${geofence:pipe}' = '-1' OR geofence_id in ($geofence))\nORDER BY start_date;",
```
Comment (adriankumpf) [inline grafana/dashboards/supercharges.json:614]: As I see, "supercharges" are determined by the charging speed:

```SQL
(charge_energy_added * 60 / NULLIF (duration_min, 0)) > 20
```

Due to the name of the dashboard I would have expected that only SuperCharges and no charges at other fast chargers would be displayed. If we wanted to include only the former we could filter for charging sessions where `fast_charger_brand` is `Tesla`.

Comment (Dulanic) [reply grafana/dashboards/supercharges.json:614]: As I think about this.... maybe I will think over a ability to use fast and/or super chargers and then set a filter. I'll work on it this weekend. Maybe modifying the charging dashboard is the best route too... I had thought about it too.

**Comment by Dulanic** [comment]
Sp I have a small dataset... do you think just setting it a filter for fast_charger_brand or fast_charger_present is best? I've never used any fast chargers outside of tesla....

**Comment by adriankumpf** [comment]
> Sp I have a small dataset... do you think just setting it a filter for fast_charger_brand or fast_charger_present is best? I've never used any fast chargers outside of tesla....

Yup, that sounds good to me. A filter for charger power would be interesting as well. We could also postpone this and create a new PR for it :)

**Comment by Dulanic** [comment]
Yeah I was thinking any fast charger is > 300 v. Maybe that's the simplistic way to do it. I'll close it this PR until then.

**Pull request closed without merging**